client/src/main/kotlin/io/spine/examples/chatspn/desktop/App.kt

```diff
@@ -46,20 +52,23 @@ public fun App() {
     var page by remember { mutableStateOf(Page.REGISTRATION) }
```

Comment:
I would eventually move the navigation out of `App.kt` into its own file.

That would also allow to hide `...Model` types from being public.